Makefile

```diff
@@ -7,7 +7,11 @@ LINUX:="Linux"
 all: parser.go
 
 test: parser.go
+	mv go.mod1 go.mod
+	mv go.sum1 go.sum
 	GO111MODULE=on go test ./...
+	mv go.mod go.mod1
+	mv go.sum go.sum1
```

Comment:
how about go test failured situation?  it seem re-run go test will meet error.

Reply:
done